lib/core/include/irods/connection_pool.hpp

```diff
@@ -15,6 +16,36 @@
 
 namespace irods
 {
+    /// Defines various options for changing the behavior of a connection pool.
+    ///
+    /// Refresh options are not mutually exclusive.
+    ///
+    /// It is important to understand that any of the options can trigger a connection
+    /// refresh. When a refresh occurs, all state related to connection refresh is reset.
+    /// Therefore, care must be taken when setting multiple options in order to obtain
+    /// deterministic refresh behavior.
+    ///
+    /// \since 4.3.1
+    struct connection_pool_options
+    {
+        /// Defines the number of times a connection can be retrieved from the pool before
```

Comment:
is it 'can be retrieved' or 'is retrieved' ?

Reply:
It's "can be retrieved" unless I'm misunderstanding the question.

So, for example ... If I set the option to 3, the pool will allow **the same connection** to be retrieved via a call to `get_connection()` 3 times before refreshing it.

Reply:
right, but... the pool won't refresh it until it hits 3?

Reply:
Will go with "is retrieved".

Reply:
done